wallet-connect/.gitignore

```diff
@@ -0,0 +1,33 @@
+node_modules
+/dist
+*.log
+.env
+
+.yarn/cache
+.yarn/install-state.gz
+
+.cache
+.eslintcache
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+
+# Editor directories and files
+.idea
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
+node_modules
+
+# macOS
+.DS_Store
```

Comment:
we already have most of this stuff in the root .gitignore. please unify

Reply:
Done